src/plugins/intel_cpu/src/nodes/mkldnn_interpolate_node.cpp

```diff
@@ -2038,20 +2125,51 @@ void MKLDNNInterpolateNode::prepareParams() {
     if (getOutputShapeAtPort(0).getRank() > 2 && (dataScales[0] != 1.f || dataScales[1] != 1.f)) {
         IE_THROW() << "Interpolate layer only supports resize on spatial dimensions(depth, height and width)";
     }
-    if ((interpAttrs.mode == InterpolateMode::nearest || interpAttrs.mode == InterpolateMode::linear_onnx || interpAttrs.mode == InterpolateMode::cubic) &&
-        ((interpAttrs.layout != InterpolateLayoutType::planar && mayiuse(cpu::x64::sse41)) ||
-            (mayiuse(cpu::x64::avx2) && interpAttrs.inPrc == Precision::FP32))) {
-        execPtr = std::make_shared<InterpolateJitExecutor>(interpAttrs,
-                                                           srcDims,
-                                                           dstDims,
-                                                           dataScales,
-                                                           attr);
-    } else {
-        execPtr = std::make_shared<InterpolateRefExecutor>(interpAttrs,
-                                                           srcDims,
-                                                           dstDims,
-                                                           dataScales);
-    }
+
+    InterpolateKey key = {interpAttrs, srcDims, dstDims, dataScales, attr};
+
+    postOpsDataPtrs.clear();
+    auto &postOps = (*key.attr.get()).post_ops_;
+    for (int i = 0; i < postOps.len(); ++i) {
+        auto &postOp = postOps.entry_[i];
+        if (postOp.is_quantization()) {
+            auto &data = postOp.quantization.data;
+            postOpsDataPtrs.insert(postOpsDataPtrs.end(), std::begin(data), std::end(data));
+            memset(data, 0, sizeof(data));
+        } else if (postOp.is_depthwise()) {
+            auto &weights = postOp.depthwise.weights_data;
+            auto &biases = postOp.depthwise.biases_data;
+            postOpsDataPtrs.push_back(weights);
+            postOpsDataPtrs.push_back(biases);
+            weights = 0;
+            biases = 0;
+        }
+    }
+
+    auto buildExecutor = [&](const InterpolateKey& key) -> std::shared_ptr<InterpolateExecutor> {
+        std::shared_ptr<InterpolateExecutor> exector;
+        if ((key.nodeAttrs.mode == InterpolateMode::nearest || key.nodeAttrs.mode == InterpolateMode::linear_onnx ||
+            key.nodeAttrs.mode == InterpolateMode::cubic) &&
+            ((key.nodeAttrs.layout != InterpolateLayoutType::planar && mayiuse(cpu::x64::sse41)) ||
+                (mayiuse(cpu::x64::avx2) && key.nodeAttrs.inPrc == Precision::FP32))) {
+            exector = std::make_shared<InterpolateJitExecutor>(key.nodeAttrs,
+                                                               key.srcDims,
+                                                               key.dstDims,
+                                                               key.dataScales,
+                                                               attr);
```

Comment:
It seems more logical to use attr form the key, and the `attr` interpolate node data member can probably be removed as useless.

Reply:
make sense, done, thanks Maksim.